metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -820,6 +835,50 @@ void main() {
       },
     );
 
+    test(
+      ".filterByProjectName() updates the current page parameters with the project name filter value",
+      () {
+        const expectedPageParameters = DashboardPageParametersModel(
+          projectFilter: projectNameFilter,
+        );
+
+        final listener = expectAsyncUntil0(
+          () {},
+          () {
+            return projectMetricsNotifier.pageParameters ==
+                expectedPageParameters;
+          },
+        );
+
+        projectMetricsNotifier.addListener(listener);
+        projectMetricsNotifier.filterByProjectName(projectNameFilter);
+      },
+    );
+
+    test(
+      ".filterByProjectName() does not update the current page project group id parameter",
```

Comment:
✅ 
Should we test that this method sets an empty string to the project filter if the given project name filter is null?